cmd/gcs/main.go

```diff
@@ -178,7 +178,7 @@ func main() {
 		uint(kmsg.Warning),
 		"Log all kmsg entries with a priority less than or equal to the supplied level.")
 	logFile := flag.String("logfile",
-		"",
+		"/tmp/gcs.log",
```

Comment:
Committed by error. Partly this is due to the difficulty in getting a useful log out. Remind me of the magic required tp capture this logging on the host side? I tried wpa etc...

Reply:
wpa using the `ContainerPlatform.wprp` file in our package should do it, as will TV++ using the adjacent `ContainerPlatform.tvpp` file
You can also create test cases in `test/functional` or `test/gcs` and run those, since they should output to stdout by default